tests/test_SLRealizer.py

```diff
@@ -2,23 +2,27 @@
 Example unit tests for SLRealizer package
 """
 import unittest
+import om10
 import desc.slrealizer
 
 class SLRealizerTestCase(unittest.TestCase):
     def setUp(self):
-        self.message = 'Hello, world'
+        self.db = om10.DB()
+        self.db.paint(synthetic=True)
 
     def tearDown(self):
         pass
 
-    def test_run(self):
-        foo = desc.slrealizer.SLRealizer(self.message)
-        self.assertEquals(foo.run(), self.message)
+    def test_init(self):
```

Comment:
It's a good idea to add more tests as we go. Whenever we find a bug, we should first write a test that causes the error message or wrong results, and fails as a result. Then, we fix the code so that the test passes. This way, we will build up a suite of tests that all need to pass when we run `nosetests`, so that problems we solve don't ever return (or if they do, we know about them).